cantools/database/can/formats/dbc.py

```diff
@@ -420,6 +435,59 @@ def long_to_short_names_dict(self):
         return self._long_to_short_names_dict
 
 
+class UniqueNamesDict(object):
+    """Create a dict with short unique names for the given list.
+
+    Map the objects' original names (short or long) to unique names with
+    up to 32 chars.
+    Use last five chars ('_0000' to '_9999' for unique enumaration of messages
+    with identical first 32 chars.
+    Skip numbers that are already used by a message in its original name with
+    exactly 32 chars.
+
+    """
+
+    def __init__(self, names):
+        self._result_dict = {}
+
+        num_objs = len(names)
+        if num_objs == 0:
+            return
```

Comment:
Remove this if-statement as it is not needed.

Reply:
yes, corrected.